source/uwp/Renderer/lib/AdaptiveCardRendererComponent.cpp

```diff
@@ -107,6 +107,7 @@ namespace winrt::AdaptiveCards::Rendering::Uwp::implementation
                     ::AdaptiveCards::Rendering::Uwp::XamlBuilder::BuildXamlTreeFromAdaptiveCard(adaptiveCard,
                                                                                                 *renderContext,
                                                                                                 m_xamlBuilder.get());
+                xamlTreeRoot.Resources().MergedDictionaries().Append(winrt::XamlControlsResources());
```

Comment:
we should not do this in the library. instead, we should make app to do it.

Can you connect with react-native-windows team to see how do they achieved this?
https://github.com/microsoft/react-native-windows

Reply:
@carlos-zamora @licanhua not sure how react-native did it but in our case I assume we will want to provide some guidance on how to add `WinUI2` resources to the dictionaries we're using :)